deprish_to_eia.py

```diff
@@ -239,33 +282,41 @@ def generate_depreciation_matches(file_path_mul,
             f"File does not exist: {file_path_deprish}"
             "Depretiation file must exist."
         )
-
-    deprish_match = match_deprish_eia(
+    deprish_match_df, possible_matches_mul_df = match_deprish_eia(
         file_path_mul, file_path_deprish,
-        sheet_name_deprish=sheet_name_deprish)
-
-    save_to_workbook(deprish_match, file_path=file_path_deprish,
-                     sheet_name=sheet_name_output)
-    return deprish_match
+        sheet_name_deprish=sheet_name_deprish,
+        sheet_name_output=sheet_name_output
+    )
+    sheets_df_dict = {sheet_name_output: deprish_match_df,
+                      "Subset of Master Unit List": possible_matches_mul_df}
+    save_to_workbook(file_path=file_path_deprish,
+                     sheets_df_dict=sheets_df_dict)
+    return deprish_match_df
 
 
-def save_to_workbook(df, file_path, sheet_name):
+def save_to_workbook(file_path, sheets_df_dict):
     """
     Save a dataframe to an existing workbook.
 
     Args:
         df (pandas.DataFrame): the table you want to export.
-        file_path (path-like): the location of the excel workbook.
-        sheet_name (string): the name of your new sheet. If this sheet exists
-            in your workbook, the new sheet will be sheet_name_1.
```

Comment:
I removed the wrong arg here (i removed file_path when i should have removed df). I'll change this..

Reply:
Note that `path-like` / `os.PathLike` means something specific, and I think it's different than what you intend here -- you're calling `file_path.exists()` below which I think means you're requiring `file_path` to be a `pathlib.Path` object, while a "path-like" argument can be either a string representation of a path, or a Path object (which knows how to represent itself as a string when required).